horovod/test_horovod.py

```diff
@@ -26,6 +27,8 @@ def _submit_spark_job(self, script):
   def test_horovod_cpu(self, configuration, controller):
     if self.getImageOs() == 'rocky':
       self.skipTest("Not supported in Rocky Linux-based images")
+    if self.getImageVersion() > pkg_resources.parse_version("2.0"):
```

Comment:
What is the reason that it's not supported on newer images?  I think the job we're trying to do here is to ensure that it *does* work on newer versions.  Horovod was last updated two weeks ago, so it's getting regular maintenance.  Let's see what we need to install in order to get this working.